precompiles/outposts/osmosis/types.go

```diff
@@ -47,6 +48,22 @@ type EventSwap struct {
 	Receiver string
 }
 
+// IBCConnction contains information of port and channel of an IBC connection.
+type IBCConnection struct {
+	PortID    string
+	ChannelID string
+}
+
+// NewIBCConnection return a new instance of IBCConnection.
+func NewIBCConnection(
+	portID, channelID string,
+) IBCConnection {
+	return IBCConnection{
+		PortID:    portID,
+		ChannelID: channelID,
+	}
+}
```

Comment:
Because reduces the number of input to functions grouping together variable that are always used together. We also have channel and port for both osmosis and evmos and this make cleaner the code